lib/msf/core/exploit/remote/kerberos/service_authenticator/base.rb

```diff
@@ -966,6 +966,8 @@ def load_credential_from_file(file_path, options = {})
   def loot_info(options = {})
     info = []
 
+    # The API here won't be able to support arbitrary search filters? i.e. searching for only realm and username
```

Comment:
TODO: This PR is in draft until the persisting/fetching of ccache objects is consolidated in a centralized place

Reply:
This was fixed in https://github.com/rapid7/metasploit-framework/pull/17377